### ISSUE TITLE
npm install fails if there is no git installed. 

### ISSUE THREAD

**Issue by anandanand84** [notifme/notifme-sdk#56]
This is caused by a dependency in node-pushnotifications -> wns

Avoid using git dependencies and projects using git dependencies

```
npm WARN addRemoteGit     at getNotFoundError (/usr/lib/node_modules/npm/node_modules/which/which.js:13:12)
npm WARN addRemoteGit     at F (/usr/lib/node_modules/npm/node_modules/which/which.js:68:19)
npm WARN addRemoteGit     at E (/usr/lib/node_modules/npm/node_modules/which/which.js:80:29)
npm WARN addRemoteGit     at /usr/lib/node_modules/npm/node_modules/which/which.js:89:16
npm WARN addRemoteGit     at /usr/lib/node_modules/npm/node_modules/which/node_modules/isexe/index.js:44:5
npm WARN addRemoteGit     at /usr/lib/node_modules/npm/node_modules/which/node_modules/isexe/access.js:8:5
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15)
npm WARN addRemoteGit  git://github.com/alex-friedl/wns.git#660afd2 resetting remote /home/trooper/.npm/_git-remotes/git-github-com-alex-friedl-wns-git-660afd2-6861c264 because of error: { Error: not found: git
npm WARN addRemoteGit     at getNotFoundError (/usr/lib/node_modules/npm/node_modules/which/which.js:13:12)
npm WARN addRemoteGit     at F (/usr/lib/node_modules/npm/node_modules/which/which.js:68:19)
npm WARN addRemoteGit     at E (/usr/lib/node_modules/npm/node_modules/which/which.js:80:29)
npm WARN addRemoteGit     at /usr/lib/node_modules/npm/node_modules/which/which.js:89:16
npm WARN addRemoteGit     at /usr/lib/node_modules/npm/node_modules/which/node_modules/isexe/index.js:44:5
npm WARN addRemoteGit     at /usr/lib/node_modules/npm/node_modules/which/node_modules/isexe/access.js:8:5
npm WARN addRemoteGit     at FSReqWrap.oncomplete (fs.js:114:15) code: 'ENOGIT' }

```

**Comment by anandanand84** [comment]
https://github.com/appfeel/node-pushnotifications/issues/84

**Comment by BDav24** [comment]
Sure, I'll update when they publish a new version.

**Comment by BDav24** [comment]
Upgrade of node-pushnotifications: https://github.com/notifme/notifme-sdk/commit/9d1c9a4b3ebd225eb095df7d7654ccd0f7c77e88#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R27
Available in v1.7.1
